ui/src/main/resources/FileManagerCode/DriveSheet.xml

```diff
@@ -2759,6 +2771,10 @@ require(['jquery', 'angular', 'JobRunner', 'angular-resource', 'angular-ui-boots
 .jobs img.avatar {
   height: 16px;
 }
+#files img.thumbnail-icon {
+  height: 32px;
+  width: 32px;
```

Comment:
This will skew the image if its aspect ratio is not 1:1. Check how the attachment thumbnails are displayed in the attachments tab. There you have a wrapper (around the image) that limits the size and centers its content. See [attachments.css](https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/viewers/attachments.css#L26).